packaging/installer/methods/packages.md

```diff
@@ -2,46 +2,78 @@
 
 ![](https://raw.githubusercontent.com/netdata/netdata/master/web/gui/images/packaging-beta-tag.svg?sanitize=true)
 
-We provide our own flavour of binary packages for the most common operating systems that comply with .RPM and .DEB
-packaging formats.
+This page covers detailed instructions on using `.deb` or `.rpm` packages to install Netdata. 
 
-We have currently released packages following the .RPM format with version
-[1.16.0](https://github.com/netdata/netdata/releases/tag/v1.16.0). We have planned to release packages following the
-.DEB format with version [1.17.0](https://github.com/netdata/netdata/releases/tag/v1.17.0). Early adopters may
-experiment with our .DEB formatted packages using our nightly releases. Our current packaging infrastructure provider is
-[Package Cloud](https://packagecloud.io).
+For certain Linux distributions (see our [distribution matrix](../../DISTRIBUTIONS.md) for supported versions and
+architectures), these packages integrate tightly with your system's package manager, making them easier to maintain,
+update, and uninstall.
 
-Netdata is committed to support installation of our solution to all operating systems. This is a constant battle for
-Netdata, as we strive to automate and make things easier for our users. For the operating system support matrix, please
-visit our [distributions](../../DISTRIBUTIONS.md) support page.
+We currently use [packagecloud](https://packagecloud.io/netdata/) to supply repositories and packages. We provide two
+separate repositories, one for nightly releases, and another for stable releases.
 
-We provide two separate repositories, one for our stable releases and one for our nightly releases.
+-   Nightly releases: [netdata/netdata-edge](https://packagecloud.io/netdata/netdata-edge)
+-   Stable releases: [netdata/netdata](https://packagecloud.io/netdata/netdata)
 
-1.  Stable releases: Our stable production releases are hosted in
-    [netdata/netdata](https://packagecloud.io/netdata/netdata) repository of package cloud
-2.  Nightly releases: Our latest releases are hosted in
-    [netdata/netdata-edge](https://packagecloud.io/netdata/netdata-edge) repository of package cloud
+Read our notice about [nightly vs. stable releases](../README.md#nightly-vs-stable-releases) to understand the
+differences between the two.
 
-Visit the repository pages and follow the quick set-up instructions to get started.
+## Quickstart
 
-## Using caching proxies with PackageCloud repositories
+packagecloud offers helper installation scripts to add their repositories to your package manager. Jump to your
+distribution and use the commands to install Netdata get _automatic nightly updates_ via your distribution's package
```

Comment:
The updates aren't technically automatic unless the user has configured their package manager to run updates automatically, so I'd suggest just saying 'nightly updates' here.